packages/sit-onyx/src/components/OnyxCheckbox/OnyxCheckbox.ct.tsx

```diff
@@ -69,28 +69,44 @@ test.describe("Screenshot tests", () => {
 
   executeMatrixScreenshotTest({
     name: "Checkbox (invalid)",
-    columns: ["unchecked", "indeterminate", "checked", "disabled", "hideLabel"],
+    columns: ["unchecked", "indeterminate", "checked", "hideLabel", "longError", "disabled"],
     rows: ["default", "hover", "focus-visible"],
-    component: (column) => (
-      <OnyxCheckbox
-        label="Test label"
-        modelValue={column === "checked"}
-        indeterminate={column === "indeterminate"}
-        hideLabel={column === "hideLabel"}
-        disabled={column === "disabled"}
-        customError="Test error"
-        value="test-value"
-      />
-    ),
+    component: (column) => {
+      const customError =
+        column === "longError"
+          ? { shortMessage: "Error", longMessage: "Further info" }
+          : "Test error";
+      return (
+        <OnyxCheckbox
+          label="Test label"
+          modelValue={column === "checked"}
+          indeterminate={column === "indeterminate"}
+          hideLabel={column === "hideLabel"}
+          disabled={column === "disabled"}
+          customError={customError}
+          value="test-value"
+        />
+      );
+    },
     beforeScreenshot: async (component, page, column, row) => {
+      if (row !== "default") {
+        // add space for tooltip
+        await component.evaluate((element) => {
+          element.style.paddingTop = "3rem";
+        });
```

Comment:
Is there a reason why you don't set this as `style` attribute above?

Reply:
I worried it might impact the tooltip, but it doesn't - all good :)